docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -623,12 +623,12 @@ USER nonroot
 CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
 ----
 
-TIP: Projects generated with https://code.quarkus.io already have this Dockerfile in the `src/main/docker` directory.
-
 Quarkus provides the `quay.io/quarkus/quarkus-distroless-image:1.0` image.
 It contains the required packages to run a native executable and is only **9Mb**.
 Just add your application on top of this image, and you will get a tiny container image.
 
+Distroless images should not be used in production without rigorous testing.
```

Comment:
Wondering if we should remove this altogether and stop building the image?

Reply:
We had requests to have one. It can still be useful even if you get a similar size (15Mb overhead) with the micro image.